test/cmd/admin.sh

```diff
@@ -42,115 +42,118 @@ status:
 
 # Ensure a removed binding gets re-added
-oc delete clusterrolebinding/cluster-status-binding
-[ ! "$(oc get clusterrolebinding/cluster-status-binding)" ]
-oadm policy reconcile-cluster-role-bindings
-[ ! "$(oc get clusterrolebinding/cluster-status-binding)" ]
-oadm policy reconcile-cluster-role-bindings --confirm
-oc get clusterrolebinding/cluster-status-binding
+os::cmd::expect_success 'oc delete clusterrolebinding/cluster-status-binding'
+os::cmd::expect_failure 'oc get clusterrolebinding/cluster-status-binding'
+os::cmd::expect_success 'oadm policy reconcile-cluster-role-bindings'
+os::cmd::expect_failure 'oc get clusterrolebinding/cluster-status-binding'
+os::cmd::expect_success 'oadm policy reconcile-cluster-role-bindings --confirm'
+os::cmd::expect_success 'oc get clusterrolebinding/cluster-status-binding'
 # Customize a binding
-oc replace --force -f ./test/fixtures/basic-users-binding.json
+os::cmd::expect_success 'oc replace --force -f ./test/fixtures/basic-users-binding.json'
 # display shows customized labels/annotations
-[ "$(oadm policy reconcile-cluster-role-bindings | grep custom-label)" ]
-[ "$(oadm policy reconcile-cluster-role-bindings | grep custom-annotation)" ]
-oadm policy reconcile-cluster-role-bindings --confirm
+os::cmd::expect_success_and_text 'oadm policy reconcile-cluster-role-bindings' 'custom-label'
+os::cmd::expect_success_and_text 'oadm policy reconcile-cluster-role-bindings' 'custom-annotation'
+os::cmd::expect_success 'oadm policy reconcile-cluster-role-bindings --confirm'
 # Ensure a customized binding's subjects, labels, annotations are retained by default
-[ "$(oc get clusterrolebindings/basic-users -o json | grep custom-label)" ]
-[ "$(oc get clusterrolebindings/basic-users -o json | grep custom-annotation)" ]
-[ "$(oc get clusterrolebindings/basic-users -o json | grep custom-user)" ]
+os::cmd::expect_success_and_text 'oc get clusterrolebindings/basic-users -o json' 'custom-label'
+os::cmd::expect_success_and_text 'oc get clusterrolebindings/basic-users -o json' 'custom-annotation'
+os::cmd::expect_success_and_text 'oc get clusterrolebindings/basic-users -o json' 'custom-user'
 # Ensure a customized binding's roleref is corrected
-[ ! "$(oc get clusterrolebindings/basic-users -o json | grep cluster-status)" ]
+os::cmd::expect_success_and_not_text 'oc get clusterrolebindings/basic-users -o json' 'cluster-status'
 # Ensure --additive-only=false removes customized users from the binding
-oadm policy reconcile-cluster-role-bindings --additive-only=false --confirm
-[ ! "$(oc get clusterrolebindings/basic-users -o json | grep custom-user)" ]
+os::cmd::expect_success 'oadm policy reconcile-cluster-role-bindings --additive-only=false --confirm'
+os::cmd::expect_success_and_not_text 'oc get clusterrolebindings/basic-users -o json' 'custom-user'
 echo "admin-reconcile-cluster-role-bindings: ok"
 
 os::cmd::expect_success "oc create -f test/extended/fixtures/roles/policy-roles.yaml"
```

Comment:
try removing the trailing `)` on lines 162 and 166:
```
os::cmd::expect_failure "oc get rolebinding/basic-users)"
...
os::cmd::expect_failure "oc get clusterrolebinding/basic-users2)"
```
and adding this after the reconcile-cluster-roles check on line 172:
```
os::cmd::expect_success "oadm policy reconcile-cluster-role-bindings --confirm"
```